resources/js/package.js

```diff
@@ -0,0 +1,8 @@
+import '/public/vendor/statamic/frontend/js/helpers.js'
```

Comment:
This is still within the base bundle, can we move this / everything to the component?

Reply:
The helpers import is required for the `window.Statamic` function I'll try to mess with it to see if I can minimize it though

Reply:
Can't we just move everything to the component?